packages/salesforcedx-vscode-soql/README.md

```diff
@@ -1 +1,25 @@
-# SOQL Builder!
+# SOQL Builder for Visual Studio Code
+
+SOQL Builder enables you to interactively build a SOQL query via a form-based visual editor. View the query as you build, and save the output to a .csv or .json file.
```

Comment:
english is hard for me.  the punctuation seems off here.  like this is a list of 3 things that should be separated by a comma instead of a full stop after editor

`SOQL Builder enables you to interactively build a SOQL query via a form-based visual editor, view the query as you build, and save the output to a .csv or .json file.
`

Reply:
yeah, I think I agree with that, I probably would have written it as one sentence with commas. @ekapner